source/proof-of-concept-guide/poc-vulnerability-detection.rst

```diff
@@ -51,12 +57,95 @@ The Vulnerability Detection module is enabled by default. You can perform the fo
 
       $ sudo systemctl restart wazuh-manager
 
-Visualize the alerts
+Testing the Configuration
+-------------------------
+
+To validate the functionality of the Vulnerability Detection module, follow these steps to ensure the system is configured correctly and the agents are communicating effectively with the manager.
+
+1. **Agent Connection**: Verify that each agent is properly connected to the Wazuh Manager. This connection is essential for transmitting system inventory data.
+
+2. **Vulnerability Scanning**:
+
+   - **Package Scanning**: The Vulnerability Detection module will automatically begin scanning the installed packages on each endpoint for known vulnerabilities.
+
+   - **OS Vulnerability Scanning**: Simultaneously, the module scans the operating system for vulnerabilities. This dual scanning approach ensures comprehensive coverage.
+
+3. **Result Visualization**:
```

Comment:
Visualization of results is what we need to show in the "Visualize the alerts" sub-section below.